sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/api/XrayWrapperArtifactoryClientTest.java

```diff
@@ -61,40 +55,37 @@ void getXrayVersion_return_version() throws JsonProcessingException {
     @Test
     void getXrayVersion_throws_xrayWrapperException() {
         /* execute + test */
-        assertThrows(XrayWrapperRuntimeException.class, () -> xrayAPIArtifactoryClient.getXrayVersion());
+        assertThrows(XrayWrapperException.class, () -> xrayAPIArtifactoryClient.getXrayVersion());
     }
 
     @Test
-    void checkArtifactoryUpload_return_true() {
+    void checkArtifactoryUpload_return_true() throws XrayWrapperException {
         /* prepare */
         Mockito.doReturn(response).when(xrayAPIArtifactoryClientSpy).send(any());
 
         /* execute + test */
-        assertTrue(xrayAPIArtifactoryClientSpy.checkArtifactoryUpload());
+        assertTrue(xrayAPIArtifactoryClientSpy.checkArtifactoryUploadSuccess());
     }
 
     @Test
-    void checkArtifactoryUpload_throws_xrayWrapperException() {
+    void checkArtifactoryUpload_throws_xrayWrapperException() throws XrayWrapperException {
         /* prepare */
-        XrayAPIResponse response = XrayAPIResponse.Builder.create(500, headers).build();
+        XrayAPIResponse response = XrayAPIResponse.Builder.builder(500, headers).build();
         Mockito.doReturn(response).when(xrayAPIArtifactoryClientSpy).send(any());
 
         /* execute + test */
-        assertThrows(XrayWrapperRuntimeException.class, () -> xrayAPIArtifactoryClient.checkArtifactoryUpload());
+        assertThrows(XrayWrapperException.class, () -> xrayAPIArtifactoryClient.checkArtifactoryUploadSuccess());
     }
 
     @Test
-    void getScanStatus_return_status_scanned() throws JsonProcessingException {
+    void getScanStatus_return_status_scanned() throws XrayWrapperException {
         /* prepare */
-        ObjectMapper mapper = new ObjectMapper();
         String jsonString = "{\"status\": \"scanned\"}";
-        response = XrayAPIResponse.Builder.create(200, headers).setBody(jsonString).build();
-        JsonNode node = mapper.readTree(jsonString);
+        response = XrayAPIResponse.Builder.builder(200, headers).addResponseBody(jsonString).build();
```

Comment:
Response implementation shall not be tested inside this class, but the artifactory client. Using the builder here tests also foreign code... couldn't we do this via mockito as done in other test methods or is this just to much effort?